### PR TITLE
Add docs

### PR DESCRIPTION
**Pull request by noblica**
Initial docusaurus setup, and some documentation added.

### CODE REVIEW

File: .gitignore-example-from-docusaurus [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.DS_Store
```
[the remark this answers is not in the document]
Comment (noblica) [reply .gitignore-example-from-docusaurus]: Yep, we don't need it. I'll move the related stuff to `.gitignore`

File: website/pages/en/users.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        </a>
+      );
+    });
+
+    return (
+      <div className="mainContainer">
+        <Container padding={['bottom', 'top']}>
+          <div className="showcaseSection">
+            <div className="prose">
+              <h1>Who's Using This?</h1>
```
Comment (jantimon) [inline website/pages/en/users.js]: Should we update this section?

File: website/pages/en/index.js [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+    .map((user, i) => {
+      return (
+        <a href={user.infoLink} key={i}>
+          <img src={user.image} alt={user.caption} title={user.caption} />
+        </a>
+      );
+    });
+
+  return (
+    <div className="productShowcaseSection paddingBottom">
```
Comment (jantimon) [inline website/pages/en/index.js]: Should we hide this section?

File: website/core/Footer.js [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+            <a href={this.pageUrl('users.html', this.props.language)}>
+              User Showcase
+            </a>
+            <a
+              href="http://stackoverflow.com/questions/tagged/"
+              target="_blank"
+              rel="noreferrer noopener">
+              Stack Overflow
+            </a>
+            <a href="https://discordapp.com/">Project Chat</a>
```
Comment (jantimon) [inline website/core/Footer.js]: Is a discord app planned?

File: website/blog/2017-09-26-adding-rss.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+---
```
Comment (jantimon) [inline website/blog/2017-09-26-adding-rss.md]: Do we have to remove the demo blog posts?

File: website/pages/en/users.js (hunk 2 above)
Comment (noblica) [reply website/pages/en/users.js]: Yeah, I think we should remove it for now, and then add it per project later.

File: website/blog/2017-09-26-adding-rss.md (hunk 5 above)
Comment (noblica) [reply website/blog/2017-09-26-adding-rss.md]: Do you want me to remove this, or...? I'm not really sure.

Comment (jantimon) [reply website/blog/2017-09-26-adding-rss.md]: I don't know docusaurus well enough - do you think we should keep them?

**Comment by jantimon** [comment]
Merged 👍👍

**Pull request closed without merging**